Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift

```diff
@@ -73,9 +97,28 @@ public final class CardPresentPaymentStore: Store {
 // MARK: - Services
 //
 private extension CardPresentPaymentStore {
+    /// Which backend is the store to use? WCPay or Stripe?
+    ///
+    enum CardPresentPaymentStoreBackend {
+        /// Use WCPay as the backend
+        ///
+        case wcpay
+
+        /// Use Stripe as the backend
+        ///
+        case stripe
+    }
+
     func startCardReaderDiscovery(siteID: Int64, onReaderDiscovered: @escaping (_ readers: [CardReader]) -> Void, onError: @escaping (Error) -> Void) {
         do {
-            try cardReaderService.start(WCPayTokenProvider(siteID: siteID, remote: self.remote))
+            switch usingBackend {
+            case .wcpay:
+                print("==== starting card reader discovery using a wcpay token provider")
+                try cardReaderService.start(WCPayTokenProvider(siteID: siteID, remote: self.remote))
```

Comment:
I would feel more comfortable merging this into trunk if we checked the feature flag in all these methods, and short-circuited defaulting to using the wcpayRemote when the feature flag is not active.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5868/commits/58187d63416200c480bdb8ab75e71be097adcbc1